### PR TITLE
Customize work directory

### PR DESCRIPTION
**Pull request by sergelogvinov**
Is simple patch allows us to customize the working directory during gh-worker registration.

Sometimes I need to change the work directory to pull/store large files on build process.

Thanks.

### CODE REVIEW

**Comment by sergelogvinov** [comment]
Hello @nikola-jokic can you take a look please...

Thanks.

File: controllers/actions.github.com/ephemeralrunner_controller.go [hunk 1]
```diff
@@ -521,6 +521,14 @@ func (r *EphemeralRunnerReconciler) updateStatusWithRunnerConfig(ctx context.Con
 	jitSettings := &actions.RunnerScaleSetJitRunnerSetting{
 		Name: ephemeralRunner.Name,
 	}
+
+	for c := range ephemeralRunner.Spec.Spec.Containers {
```
Comment (nikola-jokic) [inline controllers/actions.github.com/ephemeralrunner_controller.go]: Can you please rename `c` to `i`? c sounds like a container, while i is usually variable associated with iterations.

Comment (sergelogvinov) [reply controllers/actions.github.com/ephemeralrunner_controller.go]: Yep, thanks, fixed.

**Review by nikola-jokic — APPROVED** [review]
LGTM, thank you for your contribution, and sorry for waiting!

**Pull request merged**